.github/workflows/win.yml

```
@@ -0,0 +1,18 @@
+name: Windows
+
+on: [push, pull_request]
+jobs:
+  windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: dependencies
+        run: |
+          choco install cmake
+      - name: build
+        run: |
+          cmake -S . -B build -G "Visual Studio 15 2017" -A x64
```

Looks like only 2019 is installed. This should be it! https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#visual-studio-enterprise-2019

```suggestion
          cmake -S . -B build -G "Visual Studio 16 2019" -A x64
```